### PR TITLE
Version one

### PR DESCRIPTION
**Pull request by RoyNyaga**
This branch contains the full version of our todo-list application. We included functionalities such as: create a project, delete a project, list project, create todo, delete todo and edit the todo.

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+        "Atomics": "readonly",
+        "SharedArrayBuffer": "readonly"
+    },
+    "parserOptions": {
+        "ecmaVersion": 2018,
+        "sourceType": "module"
+    },
+    "rules": {
+    }
+}
```
Comment (jofer86) [inline .eslintrc.json]: Let's make sure we follow the [Microverse linters repo for javascript](https://github.com/microverseinc/linters-config/tree/master/javascript) instructions to set up our linters properly, tend to that, lint whatever issues that may arise from the proper configuration of eslint and stickler, and you should be golden.

**Review by jofer86 — CHANGES_REQUESTED** [review]
## :wave: :wave: Hello!, Jorge here,  I'm here to review this Pull Request, let's take a look shall we :eye: :eye: :eye: 
- Changes marked as **[OPTIONAL]** are not hard requirements but nice to have.
- The project meets all of the requirements, code is modulated properly the single responsibility principle and the module pattern is well presented on your project, just one change is necessary before this project is approved.
- Once the change below is addressed please do ask for another code review.
--------------------------------------------------------------------
Cheers, 
Jorge Fernando Rincón Figueroa.
Technical Support Engineer.
**Slack: @jofer86**

File: package.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  },
+  "homepage": "https://github.com/BenjaminGarza/to-do-list-js#readme",
+  "devDependencies": {
+    "babel-eslint": "^10.0.3",
+    "eslint": "^6.8.0",
+    "eslint-config-airbnb": "^18.0.1",
+    "eslint-config-airbnb-base": "^14.0.0",
+    "eslint-plugin-import": "^2.20.1",
+    "eslint-plugin-jsx-a11y": "^2.2.3",
+    "eslint-plugin-react": "^7.18.3",
```
Comment (jofer86) [inline package.json]: We are not using react in this project, please do remove the dev dependencies related to react and follow the configuration stated in the comment above.

**Review by jofer86 — CHANGES_REQUESTED** [review]

**Comment by RoyNyaga** [comment]
Used the microverse eslint config file and uninstalled react dependency as demanded by previous reviewer.

**Review by shloch — APPROVED** [review]
accomplished
GOOD TO GO

Happy coding .... !

**Comment by RoyNyaga** [comment]
yyyyyyuuuuuuuppppppiiiiiii thanks sir

**Pull request merged**